src/impl.ts

```diff
@@ -298,6 +298,23 @@ export async function makePostSwapCalc(
   console.log(`DUSD increase on address: ${dUsdToTransfer}`);
 }
 
+export async function burnLeftOverDFI(
+  cli: DfiCli,
+  ctx: Awaited<ReturnType<typeof createContext>>
+) {
+  const { emissionsAddr } = ctx;
+  const { balanceTokenDfi } = ctx.state.postSwapCalc;
```

Comment:
@prasannavl this should use the remaining DFI before the swap, otherwise you burn also the emission that came in during the swap (if block burst happens, this can be a lot).
should be `balanceTokensInitDfi - dfiToSwapForDiffBlocks`